README.md

```diff
@@ -134,6 +135,40 @@ Separate each step of the task to an individual task. For example a "generate ic
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Prefix tasks to bundle them
+
+Bundle your tasks with a prefix so you can execute them all at once.
+
+### Why?
+
+* Bundling helps keeping your tasks organized.
+* They are easily executed at once.
+* Your execution command stays they same when you add tasks.
```

Comment:
Rephrased it to: "Your script API remains unchanged when tasks are added, removed or renamed". Changing subtasks could also mean changing the implementation of the subtask which has nothing to do with the naming.